pom.xml

```diff
@@ -65,6 +65,8 @@
   </scm>
 
   <properties>
+    <jenkins.version>1.625.3</jenkins.version>
+    <java.level>7</java.level>
```

Comment:
Almost forgot these! Fortunately https://github.com/jenkinsci/plugin-pom/pull/88 kicked in.

Reply:
[Will be going](https://groups.google.com/d/msg/jenkinsci-dev/1xqwDRck-zk/0Fi9Jw82AAAJ) to `2.60.3` and `8` but we can merge this ahead of the bump